src/fhirtypes/common.ts

```diff
@@ -40,6 +40,28 @@ import { buildSliceTree, calculateSliceTreeCounts } from './sliceTree';
 import { InstanceExporter } from '../export';
 import { MismatchedTypeError } from '../errors';
 
+// List of Conformance and Terminology resources from http://hl7.org/fhir/R4/resourcelist.html
+// and http://hl7.org/fhir/5.0.0-snapshot1/resourcelist.html
```

Comment:
We should change this to reference the official R5 resource list at https://hl7.org/fhir/R5/resourcelist.html

Reply:
Good catch! Updated in [f4b8e27](https://github.com/FHIR/sushi/pull/1329/commits/f4b8e27f3ab1dcee1fbb315661be1ab052b7eb00).